source/ways-of-working/common-pipeline/publishing-libraries/index.html.md.erb

```diff
@@ -0,0 +1,11 @@
+---
+title: Publishing Libraries
+last_reviewed_on: 2021-10-18
+review_in: 6 months
+weight: 2
```

Comment:
can you see if you can adjust this so that the common pipeline comes before this in the side bar?